en/docs/install.html

```diff
@@ -15,6 +15,7 @@
 {% capture windows      %}{% include_relative _installations/windows.md      %}{% endcapture %}
 {% capture linux        %}{% include_relative _installations/linux.md        %}{% endcapture %}
 {% capture alternatives %}{% include_relative _installations/alternatives.md %}{% endcapture %}
+{% capture nightly      %}{% include_relative _installations/nightly.md      %}{% endcapture %}
```

Comment:
It doesn't need to be a separate page, but maybe it's own section on the same page. I don't know how well "nightly" is going to translate and I don't want people installing it because they weren't able to use one of the other options